pkg/commands/env.go

```diff
@@ -137,12 +137,14 @@ func DeleteEnv(ctx context.Context, args []string, ioStreams cmdutil.IOStreams)
 	if len(args) < 1 {
 		return fmt.Errorf("you must specify env name for vela env delete command")
 	}
-	envName := args[0]
-	msg, err := oam.DeleteEnv(envName)
-	if err == nil {
+	for _, envName := range args {
+		msg, err := oam.DeleteEnv(envName)
+		if err != nil {
+			return err
+		}
 		ioStreams.Info(msg)
```

Comment:
would you mind to refine the output message？It will output multiple times now

Reply:
You mean to display it in one line?  like:
`test1, test2 deleted `

I see that kubectl also displays multiple lines:
```
pod "xxx" deleted
pod "xxxx" deleted
```

Is it necessary to merge together？

Reply:
Do you mean to combine and output? like 
```
test1,test2 deleted.
```

Reply:
yeah, it's fine

Reply:
Is it necessary? I see that the kubectl command is also multi-line output, like:
```
pod "pod-1" deleted
pod "pod-2" deleted
```

Of course I can also change the output into one line?

Reply:
Well, it also makes sense, let me merge, thanks again !